easybuild/easyconfigs/m/mympingpong/mympingpong-0.7.1-intel-2016b-Python-2.7.12.eb

```diff
@@ -0,0 +1,33 @@
+easyblock = 'PythonPackage'
+
+name = 'mympingpong'
+version = '0.7.1'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://github.com/hpcugent/mympingpong'
+description = """A mpi4py based random pair pingpong network stress test."""
+
+toolchain = {'name': 'intel', 'version': '2016b'}
+
+source_urls = [PYPI_SOURCE]
+sources = [SOURCE_TAR_GZ]
+
+vsc_base_ver = '2.5.1'
+dependencies = [
+    ('Python', '2.7.12'),
+    ('vsc-base', vsc_base_ver, versionsuffix),
+    ('vsc-mympirun', '3.4.3', versionsuffix + '-vsc-base-%s' % vsc_base_ver),
```

Comment:
use existing `vsc-mympirun` easyconfig without `vsc-base` in suffix to resolve this dep